examples/getstarted/plugins/myplugin/admin/src/index.js

```diff
@@ -17,7 +14,11 @@ export default {
         id: `${pluginId}.plugin.name`,
         defaultMessage: 'My plugin',
       },
-      Component: App,
+      Component: async () => {
+        const component = await import(/* webpackChunkName: "my-plugin" */ './pages/App');
```

Comment:
Not sure we want a chunk starting with `my-plugin` I guess 🤔

Reply:
That's for the example plugin in the get started app 👍🏽 .